.jscsrc

```diff
@@ -0,0 +1,37 @@
+{
+  "excludeFiles": ["node_modules/**", "bower_components/**"],
+  "validateIndentation": 2,
+  "disallowMixedSpacesAndTabs": true,
+  "disallowMultipleSpaces":true,
+  "requireSemicolons":true,
+  "requireSpaceAfterLineComment":true,
+  "disallowTrailingComma":true,
+  "requireMultipleVarDecl":true,
+  "disallowSpaceAfterPrefixUnaryOperators":["++", "--", "+", "-", "~", "!"],
+  "requireLineFeedAtFileEnd":true,
+  "validateParameterSeparator":", ",
+  "requireSpaceAfterBinaryOperators":["="],
+  "requireSpaceBeforeBinaryOperators":["="],
+  "requireSpacesInsideParentheses":"all",
+  "requireSpacesInsideArrayBrackets": "all",
+  "disallowSpaceBeforeBinaryOperators":[","],
+  "requireSpacesInAnonymousFunctionExpression":{
+    "beforeOpeningCurlyBrace": true
+  },
+  "requireSpacesInCallExpression":true,
+  "requireSpacesInConditionalExpression":true,
+  "requireSpacesInForStatement":true,
+  "requireSpacesInFunctionDeclaration":{
+    "beforeOpeningCurlyBrace": true
+  },
+  "requireSpacesInFunctionExpression":{
+    "beforeOpeningCurlyBrace": true
+  },
+  "requireSpacesInFunction":{
+    "beforeOpeningCurlyBrace": true
+  },
+  "requireSpacesInNamedFunctionExpression":{
+    "beforeOpeningCurlyBrace": true
+  },
+  "requireDollarBeforejQueryAssignment":true
```

Comment:
spacing: `"requireDollarBeforejQueryAssignment": true`. Same thing for some other rules in this file.